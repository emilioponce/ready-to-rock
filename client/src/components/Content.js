import React from 'react'
import BandsBrowser from './BandsBrowser'

/* Now Content only invokes one component. It can be easily extended if it's necessary */
const Content = () => {
  return <BandsBrowser />
}

export default Content
