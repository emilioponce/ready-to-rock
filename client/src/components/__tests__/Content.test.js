import React from 'react'
import ReactDOM from 'react-dom'
import Content from '../Content'
import { shallow } from 'enzyme'

const component = <Content />

describe('Component Content', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div')
    ReactDOM.render(component, div)
    ReactDOM.unmountComponentAtNode(div)
  })

  it('snapshot renders correctly', () => {
    const wrapper = shallow(component)
    expect(wrapper).toMatchSnapshot()
  })
})
