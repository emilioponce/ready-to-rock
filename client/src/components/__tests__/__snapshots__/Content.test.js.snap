// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component Content snapshot renders correctly 1`] = `<BandsBrowser />`;
